code/modules/mob/mob.dm

```diff
@@ -618,6 +619,9 @@
 				stat("CPU:","[world.cpu]")
 				stat("Instances:","[world.contents.len]")
 				stat(null, "Time Dilation: [round(SStime_track.time_dilation_current,1)]% AVG:([round(SStime_track.time_dilation_avg_fast,1)]%, [round(SStime_track.time_dilation_avg,1)]%, [round(SStime_track.time_dilation_avg_slow,1)]%)")
+				stat("Keys Held", keys2text(client.move_keys_held | client.mod_keys_held))
```

Comment:
Why is this being thrown into the status panel? Won't that be a fair bit of extra overhead for no useful purpose?